source/task_scheduler.cpp

```diff
@@ -335,13 +351,21 @@ int TaskScheduler::Init(TaskSchedulerInitOptions options) {
 		}
 	}
 
+	// Manually invoke for 'main' fiber
+	if (m_callbacks) {
+		m_callbacks->OnWorkerThreadStarted(0);
+		m_callbacks->OnFiberStateChanged(GetCurrentFiberIndex(), ISchedulerUserCallbacks::FiberState::ATTACHED);
```

Comment:
Minor. But the main fiber is always index zero (we assign that above). So we can replace the `GetCurrentFiberIndex()` call with just `0`